src/app/themes/Theme.ts

```diff
@@ -6,21 +6,34 @@ export const theme = createTheme({
       main: '#646464',
     },
     secondary: {
-      main: '#5c5c5c',
+      main: '#35aed7',
     },
   },
   props: {
     MuiButton: {
-      variant: 'contained',
-      color: 'primary',
+      color: 'secondary',
+      variant: 'outlined',
     },
   },
   overrides: {
     MuiButton: {
       root: {
-        borderRadius: '5rem',
+        backgroundColor: 'black',
         textTransform: 'none',
-        width: '6rem',
+        width: '10rem',
+        padding: '2rem',
+        border: 'solid 2px',
+        fontSize: '1.2rem',
+      },
+      contained: {
+        border: 'none',
+        borderRadius: 0,
+        color: 'white !important',
```

Comment:
WDYT about try to make it work without`!important`? is it necessary?

Reply:
It's not needed. I just forgot to remove it :D